### PR TITLE
Admin edit item: Closes #46

### LINKED ISSUE #46: 25: Admin edits an item
As an admin 
When I visit "admin/items" 
And I click "Edit" 
Then my current path should be "/admin/items/:ITEM_ID/edit" 
And I should be able to update title, description, image, and status

### PR DESCRIPTION
**Pull request by MikelSage**
Admin can now edit items

### CODE REVIEW

File: app/assets/stylesheets/admin/items.scss [hunk 1]
```diff
@@ -1,3 +1,13 @@
 // Place all the styles related to the Admin::Items controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.opaque-back {
+  background-color: rgba(255, 255, 255, 0.5);
+  padding: 30px;
+  margin-bottom: 30px;
+}
```
Comment (swdonovan) [inline app/assets/stylesheets/admin/items.scss:9]: This is awesome and makes the site styling look a hell of a lot better

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -9,7 +9,11 @@ def create
     if @user && @user.authenticate(params[:session][:password])
       session[:user_id] = @user.id
       flash[:success] = 'Logged in successfully'
-      redirect_to dashboard_path(id: @user.id)
+      if current_admin
+        redirect_to admin_dashboard_path
+      else
+        redirect_to dashboard_path(id: @user.id)
+      end
```
Comment (swdonovan) [inline app/controllers/sessions_controller.rb:16]: nice work!!

Comment (jimszalew) [inline app/controllers/sessions_controller.rb:16]: Nice work here splitting out the redirects.

File: app/views/admin/items/new.html.erb [hunk 3]
```diff
@@ -1,22 +1,3 @@
...
-      <%= c.label {c.check_box(type: 'checkbox') + c.text} %>
-    </div>
-  <% end %>
-
-  <%= f.label :image %>
-  <%= f.file_field :image %>
-
-  <%= f.submit %>
-<% end %>
+<%= render partial: 'form' %>
```
Comment (swdonovan) [inline app/views/admin/items/new.html.erb:3]: love partials!

**Review by swdonovan — APPROVED** [review]

File: app/models/item.rb [hunk 4]
```diff
@@ -4,6 +4,7 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
   has_many :order_items
   has_attached_file :image, styles: { small: "64x64", med: "100x100", large: "200x200"}, default_url: "http://static.neatorama.com/images/2014-04/bazooka-vespa.jpg"
+  validates_attachment_content_type :image, content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"]
```
Comment (jimszalew) [inline app/models/item.rb:7]: This is slick. Didn't know you could pass it an array of acceptable attachment types.

File: app/views/admin/items/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    <%= collection_check_boxes(:item, :category_ids, Category.all, :id, :name) do |c| %>
+      <div class="checkbox" style="height: 10%; width: 33%; display: inline-block;">
+        <%= c.label {c.check_box(type: 'checkbox') + c.text} %>
+      </div>
+    <% end %>
+
+    <div class="form-group">
+      <%= f.label :image %>
+      <%= f.file_field :image %>
+    </div>
```
Comment (jimszalew) [inline app/views/admin/items/_form.html.erb:28]: I really like the use of css class here. Very versatile.

File: app/views/layouts/_navbar.html.erb [hunk 6]
```diff
@@ -30,7 +30,7 @@
           <ul class="dropdown-menu acct-dropdown-menu">
             <% if current_user %>
               <% if current_admin %>
-                <li><a href="<%=  admin_user_path(current_user.id)  %>">Admin Dashboard</a></li>
+                <li><a href="<%=  admin_dashboard_path  %>">Admin Dashboard</a></li>
```
Comment (jimszalew) [inline app/views/layouts/_navbar.html.erb:33]: This makes way more sense. Nice modification.

**Review by jimszalew — APPROVED** [review]
This all looks legit. Good job

**Pull request merged**